api_moq_test.go

```diff
@@ -335,3 +347,34 @@ func (mock *APIMock) SearchCheckBundlesCalls() []struct {
 	mock.lockSearchCheckBundles.RUnlock()
 	return calls
 }
+
+// UpdateCheckBundle calls UpdateCheckBundleFunc.
+func (mock *APIMock) UpdateCheckBundle(cfg *apiclient.CheckBundle) (*apiclient.CheckBundle, error) {
+	if mock.UpdateCheckBundleFunc == nil {
+		panic("APIMock.UpdateCheckBundleFunc: method is nil but API.UpdateCheckBundle was just called")
+	}
+	callInfo := struct {
```

Comment:
Should this type (struct{*apiclient.CheckBundle}) be defined somewhere, since it is used in multiple places? Maybe not if these are only used in a test.

Reply:
apiclient.CheckBundle is defined in circonus-labs/go-apiclient